apis/supervisor/config/v1alpha1/types_federationdomain.go.tmpl

```diff
@@ -58,6 +207,32 @@ type FederationDomainSpec struct {
 	// TLS configures how this FederationDomain is served over Transport Layer Security (TLS).
 	// +optional
 	TLS *FederationDomainTLSSpec `json:"tls,omitempty"`
+
+	// IdentityProviders is the list of identity providers available for use by this FederationDomain.
+	//
+	// An identity provider (e.g. OIDCIdentityProvider or LDAPIdentityProvider) describes how to connect to a server,
+	// how to talk in a specific protocol for authentication, and how to use the schema of that server/protocol to
+	// extract a normalized user identity. Normalized user identities include a username and a list of group names.
+	// In contrast, IdentityProviders describes how to use that normalized identity in a group of
+	// Kubernetes clusters. It can perform arbitrary transformations on that normalized identity. For example, a
```

Comment:
`It can perform arbitrary transformations on that normalized identity.` 
It being the `IdentityProviders` in the way I'm reading this. That is a little confusing in that it is each individual `FederationDomainIdentityProvider` that will have its own separate set of transformers on the identities.  There isn't a "transformer list" that works on the set of `[]IdentityProviders`.

Reply:
Will update to try to clarify.